lang/validate.go

```diff
@@ -25,3 +25,11 @@ func (dm DiagnosticsMap) Extend(diagMap DiagnosticsMap) DiagnosticsMap {
 
 	return dm
 }
+
+func (dm DiagnosticsMap) Count() int {
```

Comment:
```suggestion
// Count returns the number of diagnostics for all files
func (dm DiagnosticsMap) Count() int {
```

This is just to avoid any doubts about whether this is possibly an equivalent of `len(DiagnosticsMap)`, i.e. number of entries in the map, i.e. files.